src/components/form/Form.js

```diff
@@ -234,6 +234,9 @@ function Form({route}) {
 												),
 											},
 											method: 'POST',
+											headers: {
+												'Accept-Language': `${state.lang.replace("_","-")}`,
```

Comment:
state is not an available variable in this file. You would have to destructure it from useAppState(). Also `lang` on the  state object is hard coded to 'en_US'. Why would setting it to this fix the form issue?

Reply:
I modified the PR to include the state.
Regarding `lang`, I don't know what was the initial motivation to set this param, but I guess it meant the default app's language, and I guess it must be the same that your default language in Liferay. Although right now it is hardcoded, it shouldn't be too hard adding this param to settings.
I transformed it from en_US to en-US, because is the format accepted by "Accept-Language".
I added the header following the instructions within this [ticket](https://issues.liferay.com/browse/LPS-123596)